src/modules/grubcfg/main.py

```diff
@@ -43,6 +45,9 @@ def modify_grub_default(partitions, root_mount_point, distributor):
 
     if plymouth_bin == 0:
         use_splash = "splash"
+        
+        plymouth_theme = libcalamares.globalstorage.value("plymouth_theme")
```

Comment:
I think you mean `libcalamares.job.configuration`, not `globalstorage`. Also, since this is an optional configuration entry, you should check `if "plymouth_theme" in libcalamares.job.configuration` before accessing it.